portfolio/src/main/webapp/style.css

```diff
@@ -37,7 +37,7 @@ h4 {
   text-align: center;
 }
 
-p {
+p#about {
```

Comment:
Should this be div#about ? That would match the identifier on line 36.

Reply:
It also seemed like this element was not doing anything so I deleted the css - it wasn't helping anything anyways!